packages/glyphs-block/src/GlyphsBlock.tsx

```diff
@@ -25,13 +25,12 @@ export const GlyphsBlock = ({ appBridge }: BlockProps): ReactElement => {
         radiusValue,
     } = blockSettings;
 
-    const splittedItems = chars.split(',');
+    const splittedItems = chars.split(',').map((char) => char.trim());
+    const numberOfRows = Math.ceil(splittedItems.length / 6);
```

Comment:
Can we put that `6` into constant like `ITEMS_PER_ROW`

Reply:
Good point, added 👍🏻